pkg/apis/autoscaling/v1alpha1/pa_lifecycle.go

```diff
@@ -87,9 +87,11 @@ func (pa *PodAutoscaler) annotationFloat64(key string) (float64, bool) {
 func (pa *PodAutoscaler) ScaleBounds() (min, max int32) {
 	if pa.Spec.Reachability != ReachabilityUnreachable {
 		min = pa.annotationInt32(autoscaling.MinScaleAnnotationKey)
+		max = pa.annotationInt32(autoscaling.MaxScaleAnnotationKey)
+	} else {
+		// Unreachable, so let's immediately scale the revision to 1.
+		max = 1
```

Comment:
This should at least be minScale I think.  Technically a revision with minScale = 2 scaled to 1 isn't ready.

Reply:
The already _existing_ behavior is to use 0 for minScale when revision is not reachable.